sigmf/schema.py

```diff
@@ -21,10 +21,8 @@ def get_schema(version=None, schema_file=SCHEMA_META):
 
     TODO: In the future load specific schema versions.
     '''
-    schema_path = os.path.join(
-        utils.get_schema_path(os.path.dirname(utils.__file__)),
-        schema_file
-    )
+    
+    schema_path = Path.joinpath(utils.get_schema_path(Path(utils.__file__).parent), schema_file)
```

Comment:
`get_schema_path` should be different:
```python
def get_schema_path(version: str=__version__) -> Path:
    """
    TODO: Allow getting different schemas for specific SigMF versions
    """
    return Path(__file__)
```

so this line would just be `schema_path =  utils.get_schema_path() / schema_file`

Reply:
But, `get_schema_path()` take `module_path: str` as parameter.

Reply:
I wanna ask something. What version is this `__version__`.

Reply:
You can do `from . import __version__` which will import the current version string stored in `__init__.py`.

Reply:
Thank you.